### DISCUSSION TITLE
Path local Excel Files on Windows

### DISCUSSION THREAD

**Discussion by rodrigodel** [airbytehq/airbyte#34349] [Connector Questions]
How can I configure these three paths to locate Excel files locally? I've tried almost everything but I couldn't do it on Windows.
LOCAL_ROOT=C//airbyte_data/data
LOCAL_DOCKER_MOUNT=C//airbyte_data/data
HACK_LOCAL_ROOT_PARENT=C//airbyte_data

**Comment by sajarin** [comment]
To configure the paths for local Excel files on Windows, you need to modify the .env file in your local Airbyte root folder. The values for LOCAL_ROOT, LOCAL_DOCKER_MOUNT, and HACK_LOCAL_ROOT_PARENT should be set to an existing absolute path on your machine.

Here's how you should set them:

LOCAL_ROOT and LOCAL_DOCKER_MOUNT should be set to the same value, which is the absolute path where your local files are stored.
HACK_LOCAL_ROOT_PARENT should be set to the parent directory of the above path.
Please note that colons in the path need to be replaced with a double forward slash, //.

For example, if your files are stored in C:\airbytedata\data, you should set the values as follows:

LOCAL_ROOT=C//airbytedata/data
LOCAL_DOCKER_MOUNT=C//airbytedata/data
HACK_LOCAL_ROOT_PARENT=C//airbytedata
Also, when you are replicating data from a locally stored file, the local storage URL for reading must start with the local mount "/local/".

Please refer to the [Airbyte documentation](https://docs.airbyte.com/integrations/sources/file#local-filesystem-airbyte-open-source-only) for more details.

If you're still having trouble locating your files, you can follow the steps in this [guide](https://docs.airbyte.com/integrations/locating-files-local-destination#locating-where-your-temp-folder-is) to locate your temp folder when running Airbyte's Docker image on Windows with WSL2.

[a comment by an automated account is not shown]
